schema/defs/vrs/GenotypeMember.rst

```diff
@@ -0,0 +1,28 @@
+**Computational Definition**
+
+A class describing a :ref:`Genotype` `member`.
```

Comment:
More specific description would be helpful to readers I think.  Something like "A representation of the number of copies of a given  haplotype or allele that make up the members of a genotype. This structure type is only used within the Genotype data type."

We should also provide 2 examples of this so that we can emphasize how the copies is used to represent Het vs Hom alleles or haplotypes.  It will also be useful to reuse the examples in the Genotype examples so that we can clearly show the purpose of the Genotype.copies from the GenotypeMember.copies which will most certainly be a point of confusion.

We may need implementation guidance as well, but it may make sense to simply have all the implementation guidance in the Genotype section.